external/js/kyber/dist/constants.js

```diff
@@ -0,0 +1,10 @@
+"use strict";
```

Comment:
I wonder if you can use .gitattributes to mark everything in dist/... as binary, so that we don't see diffs of it.